src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/ListenerPrimary.cs

```diff
@@ -33,6 +33,9 @@ protected ListenerPrimary(ServiceContext serviceContext) : base(serviceContext)
         {
         }
 
+        // For testing purposes.
+        internal int UvPipeCount => _dispatchPipes.Count;
```

Comment:
cc @muratg since there's a src code change (even though nothing's changing 👍 )

Reply:
This looks fine to me (adding internal property in an internal namespace). @Eilon is this OK?

Reply:
Not crazy about this. I would rather not have any API change at all.